src/kakarot/library.cairo

```diff
@@ -125,7 +126,7 @@ namespace Kakarot {
 
         // Start execution
         let ctx = EVMInstructions.run(ctx);
-
+        ExecutionContext.maybe_throw_revert(ctx);
```

Comment:
This is where I refactored the logic that takes the revert bytes and gets a short string.